### DISCUSSION TITLE
How to modify the webclient?

### DISCUSSION THREAD

**Discussion by Mtlinear** [jcorporation/myMPD#366] [Q&A] [answered]
Hello, 
I would like to modify the webclient. I tried to modify the *.html files, but it has no effect. How can I modify the site ? Should I rebuild/recompile anything ?

Thank you for your help.
Mtlinear

**Comment by jcorporation** [comment, marked as answer]
The files in the htdocs directory are embedded in the executable. You must compile it after modifying the files in the htdocs directory. Compiled in debug mode the assets are not embedded and delivered directly from the htdocs directory.

Look at: https://github.com/jcorporation/myMPD/wiki/Building-myMPD

What things do you want to change?

**Comment by Mtlinear** [comment]
Thank you very much for your answer. I was able to make the changes.

It's mainly out of curiosity, to understand how the program works. But, I also have the project to add an audio player (so i don't have to have another page open with the mpd stream)

**Comment by jcorporation** [comment]
There is a local Player integrated to listen to the mpd stream. You must enable it in the settings.
